### PR TITLE
Fixes #14926 - Support for hammer plugin

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

File: app/controllers/api/v2/compliance/arf_reports_controller.rb [hunk 1]
```diff
@@ -31,7 +31,7 @@ def get_resource
         end
 
         resource_description do
-          resource_id 'foreman_openscap_arf_reports'
+          resource_id 'arf_reports'
```
Comment (ares) [inline app/controllers/api/v2/compliance/arf_reports_controller.rb]: I'm not apipie expert, but based on documentation, this should not be needed anymore? https://github.com/Apipie/apipie-rails#resource-description

@shlomizadok do you happen to know what was the reason for custom resource_id?

Comment (shlomizadok) [inline app/controllers/api/v2/compliance/arf_reports_controller.rb]: No, tbh.
I believe that all of the resource_description can be moved (from all of the controllers)

**Comment by shlomizadok** [comment]
@xprazak2 - Thanks for this pull request. 
I suggest we'll remove the `resource_description` from our API controllers. I believe apipie will play nicely without them

**Comment by xprazak2** [comment]
I removed `resource_description`, it works without it.

**Comment by xprazak2** [comment]
I also changed `scap_content/show` in api controller so we could have the `info` command in hammer. The original was renamed to `xml`.

There were failures in `AvailableProxy` related code when proxy was down/unavailable, now we should get nice message in all cases.

I added a few more nodes into rabl views as well.

**Comment by shlomizadok** [comment]
@xprazak2 - Thank you so much for this PR. Merging

**Pull request merged**